components/freertos/port/esp8266/port.c

```diff
@@ -237,17 +242,17 @@ void IRAM_ATTR vPortExitCritical(void)
                 }
             }
         } else {
-            ets_printf("E:C:%d\n", uxCriticalNesting);
+            ets_printf("E:C:%lu\n", uxCriticalNesting);
             PORT_ASSERT((uxCriticalNesting > 0));
         }
     }
 }
 
 void ShowCritical(void)
 {
-    os_printf("ShowCritical:%d\n", uxCriticalNesting);
-    os_printf("HdlMacSig:%d\n", HdlMacSig);
-    os_printf("SWReq:%d\n", SWReq);
+    os_printf("ShowCritical:%lu\n", uxCriticalNesting);
+    os_printf("HdlMacSig:%c\n", HdlMacSig);
+    os_printf("SWReq:%c\n", SWReq);
```

Comment:
Both `HdlMacSig` and `SWReq` are used as boolean values (either 0 or 1), even though a `char` is used to store it, there is no point in print it as character.

Reply:
Yes, define HdlMacSig & SWReq as uint8_t is also OK.